z/labels.py

```
@@ -13,58 +13,152 @@
+    return data_to_work
+
+
+def work_in_list(data_to_work):
+
+    same_label = 0
+    same_desc = 0
+
+    diff_label = 0
+    diff_desc = 0
+
+    for en, tab in tqdm(data_to_work.items()):
```

Ruff (0.12.2)

Loop control variable `en` not used within loop body

Rename unused `en` to `_en`

(B007)